enhancements/sig-architecture/6-placements/README.md

```diff
@@ -0,0 +1,470 @@
+# New Placement APIs
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The proposed work would define new placement APIs to select a set of managed clusters from multiple ManagedClusterSets.
+
+## Motivation
+
+Since placementrule is used in multiple components (GRC/App/Observability). It is more appropriate to  define it in API repo and have a generic component to consume it based on the current placementrule APIs in https://github.com/open-cluster-management/multicloud-operators-placementrule 
```

Comment:
nit: breaking lines on sentences makes it a little easier to comment.

Reply:
Updated.